Pythonwin/pywin/framework/dbgcommands.py

```diff
@@ -105,7 +105,9 @@ def OnAdd(self, msg, code):
             ## for the toolbar button IDC_DBG_ADD fails, since MFC falls back to
             ## sending a normal command if the UI update command fails.
             ## win32ui.MessageBox('There is no active window - no breakpoint can be added')
-            warnings.warn("There is no active window - no breakpoint can be added")
```

Comment:
this isn't really going to be actionable by the user, and like above, it's never going to be their code at fault. This is more akin to an assertion (although I don't think that's useful either).